apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -51,6 +54,46 @@ export function AlphabetCardDetailScreen() {
     //TODO pull this from a config
     const apiUrlPrefix = `http://10.0.2.2:3131/api/resources/mediaitems/download?name=`;
 
+    const letterAudio = new Sound(
+        `${apiUrlPrefix}${letter_audio.replace('.mp3', '')}`,
+        Sound.MAIN_BUNDLE,
+        (error) => {
+            if (error) {
+                console.log('failed to load letter audio', error);
+                return;
+            }
+        }
+    );
+
+    const playLetter = () => {
+        letterAudio.play((success) => {
+            if (!success) {
+                console.error('Error playing letter audio');
+            }
+        });
+    };
+
+    const wordAudio = new Sound(
```

Comment:
This is the third time I've seen the same pattern. I wonder if you should create an abstraction around this.